packages/utils/src/bytes.ts

```diff
@@ -56,6 +56,14 @@ export function toHex(buffer: Uint8Array | Parameters<typeof Buffer.from>[0]): s
   }
 }
 
+// Shared buffer to convert root to hex
+const rootBuf = Buffer.alloc(32);
+
+export function toRootHex(root: Uint8Array): string {
```

Comment:
what happens if you pass in a root > 32 bytes? I would assume it just cuts of the end but we could consider throwing a more explicit error if this function is misused